scipy/optimize/cython_optimize/__init__.py

```diff
@@ -0,0 +1,141 @@
+"""
+Cython Optimize Zeros API
+=========================
+The following four zeros functions can be accessed directly from Cython:
+
+- `~scipy.optimize.bisect`
+- `~scipy.optimize.ridder`
+- `~scipy.optimize.brenth`
+- `~scipy.optimize.brentq`
+
+The Cython API is similar to Python except there is no ``disp`` argument.
+Import the zero functions into Cython directly from ``cython_optimize``. ::
+
+    from scipy.optimize.cython_optimize cimport bisect, ridder, brentq, brenth
+
+
+Callback Signature
+------------------
+The zeros functions in this module use a callback that takes a double with the
+scalar independent variable as the 1st argument and a user defined ``struct``
+with any extra parameters as the 2nd argument. ::
+
+    double (*callback_type)(double, void*)
+
+
+Examples
+--------
+Usage of `~scipy.optimize.cython_optimize` requires Cython to write callbacks
+that are compiled into C. For more information on compiling Cython see the
+`Cython Documentation <http://docs.cython.org/en/latest/index.html>`_.
+
+These are the basic steps:
+
+1. Create a Cython ``.pyx`` file, for example: ``myexample.pyx``
+2. Import the desired root finder from `~scipy.optimize.cython_optimize`
```

Comment:
there's an odd `~` in that link, typo?

Reply:
I use the tilda to omit the extra namespaces, so it just shows up as `cython_optimize`, see [the rendered examples](https://9801-1460385-gh.circle-artifacts.com/0/html-scipyorg/optimize.cython_optimize.html#examples). It seems to work, but perhaps this is not standard practice? I can remove the tilda if preferred.